cypress/integration/manual-event-sync.spec.js

```diff
@@ -0,0 +1,158 @@
+import ManualEventSyncPageObject from "../support/page-objects/ManualEventSyncPageObject";
+/**
+ * Database: d2-docker-eyeseetea-2-30-datasync-sender
+ */
+context("Manual event sync", function() {
+    const page = new ManualEventSyncPageObject(cy);
+
+    const inputs = {
+        orgUnit: "Akrodie Health Centre",
+        orgUnitLevel1: "Ahafo",
+        orgUnitLevel2: "Asunafo North",
+        orgUnitLevel3: "Akrodie",
+        event: "PoKTdJQa8pV",
+        instance: "Y5QsHDoD4I0",
+        program: "ENTO- ",
```

Comment:
This input is ambiguous because exists many rows beginning by "ENTO-" if docker instance changes the metadata on the future, this test could select a different row and fail.

Reply:
The reason I use that name is because of a bug in the api, those programs have a space in the name after "ento-" ("ento-"), and the api returns it without double space, the filter does not work well in that case and I had to use that solution. The most logical thing would be to change the name to remove the double space or fix the problem of the api in the filter.
http://dev2.eyeseetea.com:8085/api/programs?order=displayName%3Aiasc&page=1&pageSize=25&fields=code%2Ccreated%2Cdescription%2CdisplayName%2Chref%2Cid%2ClastUpdated%2Cname%2CprogramStages%5BdisplayName%2CprogramStageDataElements%5BdataElement%5Bcode%2Ccreated%2Cdescription%2CdisplayName%2Chref%2Cid%2ClastUpdated%2Cname%2CshortName%5D%5D%5D%2CshortName&filter=name%3Ailike%3AENTO-%20%20Discriminating%20concentration%20bioassay
not works.

http://dev2.eyeseetea.com:8085/api/programs?order=displayName%3Aiasc&page=1&pageSize=25&fields=code%2Ccreated%2Cdescription%2CdisplayName%2Chref%2Cid%2ClastUpdated%2Cname%2CprogramStages%5BdisplayName%2CprogramStageDataElements%5BdataElement%5Bcode%2Ccreated%2Cdescription%2CdisplayName%2Chref%2Cid%2ClastUpdated%2Cname%2CshortName%5D%5D%5D%2CshortName&filter=name%3Ailike%3AENTO-%20Discriminating%20concentration%20bioassay
works.

Reply:
@idelcano Could you please merge with https://github.com/EyeSeeTea/metadata-synchronization/pull/391 and test now? I have changed how the search works and now your scenario shouldn't happen anymore :)